Wikipedia.xcodeproj/xcshareddata/xcschemes/Local mediawiki-services-mobileapps.xcscheme

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should we delete the Local (WIP:) scheme? Looks the same as this one.

Reply:
yup, done!